docs/paper/dev/getting-started/paper-plugins.md

```diff
@@ -0,0 +1,116 @@
+---
+slug: /dev/getting-started/paper-plugins
+---
+
+# Paper Plugins
+
+Paper plugins allow developers to take advantage of more modern concepts introduced by Mojang, such as datapacks, to 
+expand the field of what the Paper API is able to introduce.
+
+:::info Experimental
+
+This is experimental and may be subject to change.
+
+:::
+
+- [Bootstrapper](#bootstrapper)
+- [Loader](#loaders)
+- [Differences](#differences)
+
+## How do I use them?
+Similarly to Bukkit plugins, you have to introduce a ``paper-plugin.yml`` file into your jar resources folder.
+Here is an example configuration.
+```yml
+name: Paper-Test-Plugin
+version: ${version}
+main: io.papermc.testplugin.TestPlugin
+description: Paper Test Plugin
+author: PaperMC
+api-version: ${apiversion}
+load: STARTUP
+bootstrapper: io.papermc.testplugin.TestPluginBootstrap
+loader: io.papermc.testplugin.TestPluginLoader
+defaultPerm: FALSE
+permissions:
+dependencies:
```

Comment:
Maybe remove this if it's not required to be blank for the plugin to be loaded, dependencies are explained directly below it.